R/create-irods.R

```diff
@@ -1,33 +1,32 @@
-#' Generate IRODS Project File
+#' Generate IRODS Configuration File
 #'
-#' This will create an iRODS project file containing information about the
+#' This will create an iRODS configuration file containing information about the
 #' iRODS server. Once the file has been created, future sessions
 #' connect again with the same iRODS server without further intervention.
 #'
+#' The configuration file is located in the user-specific configuration
+#' directory. This destination is either set with R_USER_DATA_DIR and
+#' R_USER_CONFIG_DIR if set. Otherwise, they follow platform conventions (
```

Comment:
This paragraph is not clear to me.

Does this mean what you mean?
```
#' The configuration file is located in the user-specific configuration
#' directory, defined by either R_USER_DATA_DIR or R_USER_CONFIG_DIR.
#' Otherwise, they follow platform conventions (
```

I still would not know which one takes precedence if they're both defined.

Reply:
I see, `R_USER_DATA_DIR` does not belong there, only `R_USER_CONFIG_DIR` is relevant. I will remove  `R_USER_DATA_DIR`  from the man page.

Reply:
ah, even better.

Reply:
fixed